prototype/jurisdictions/certificate.AU.xml

```diff
@@ -86,16 +74,16 @@
 							<help>If people used their creativity or judgement to contribute data then they have copyright over their work. For example, writing a description or deciding whether or not to include some data in a dataset would require judgement. So contributors must transfer or waive their rights, or license the data to you before you can publish it.</help>
 						</question>
 						<if test="this.crowdsourcedContent() === 'true'">
-							<question id="claUrl" display="The Contributor Licence Agreement is at">
-								<label>Where is the Contributor Licence Agreement (CLA)?</label>
+							<question id="claUrl" display="The contributor licence or waiver is at">
```

Comment:
@Floppy we could make these changes across all the versions of the certificates. "Contributor License Agreement" is the recognised term but not using it might help the questionnaire to be more approachable.